src/main/java/org/idpass/lite/Card.java

```diff
@@ -47,10 +47,22 @@
     private boolean isAuthenticated = false;
     private byte[] cardPublicKey = null;
     private byte[] cardAsByte = null;
+    private boolean scaledQR = false;
 
     private HashMap<String, Object> cardDetails = new HashMap<String, Object>();
     private HashMap<String, String> cardExtras = new HashMap<String, String>();
 
+    /**
+     * Use to generate either a scaled-up QR code image or not. A scaled-up QR code image file is
+     * more challenging to read by Zxing. For test cases purposes, the default
```

Comment:
I don't think the scaled-up QR Code image is the challenge. The challenge is the way the scale-up is done. It should not add any blur to it when done. Something like nearest-neighbor scaling should be used.

Reply:
https://github.com/journeyapps/zxing-android-embedded/issues/395

Reply:
Scaling method in `IDPassReader::getQRCode()` is a direct pixel by pixel scaling.